roles/dns/templates/named.service.j2

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Requires=docker.service
+After=docker.service
```

Comment:
Probably should have the following as well:
```
PartOf=docker.service
```
To allow the service to be restarted if someone issues a systemctl docker restart.